src/editor/ImageViewer.js

```diff
@@ -121,31 +128,39 @@ define(function (require, exports, module) {
      */
     function ImageView(file, $container) {
         this.file = file;
+        this.$container = $container;
+
+        this._naturalWidth = 0;
+        this._naturalHeight = 0;
+        this.relPath = ProjectManager.makeProjectRelativeIfPossible(this.file.fullPath);
+
+        this._buildPage(file, $container);
+
+        _viewers[file.fullPath] = this;
+    }
+
+    // Updates the page markup and...
+    // * Assigns variables to elements
+    // * Adds load and error listeners
+    ImageView.prototype._buildPage = function (file, $container) {
+
         this.$el = $(Mustache.render(ImageViewTemplate, {
-            imgUrl: _getImageUrl(file),
+            imgUrl: _getImageUrl(this.file),
+            localImgUrl: _getLocalAssetUrl(this.file),
             Strings: Strings
         }));
 
         $container.append(this.$el);
```

Comment:
You'd think so, right?? But it seems to work. I'll double check though, maybe we have some weird phantom DOM that's causing the issues you've seen above. That could very well be the case actually since we're positioning these absolutely (on top of each other?)